pages/api/v1/auth/actions/reportContent.ts

```diff
@@ -83,13 +84,26 @@ async function handler(req: NextApiRequest, res: NextApiResponse<IReportContentR
 		const totalUsers = data.count || 0;
 
 		if(type == 'post' && checkReportThreshold(totalUsers) ){
-			_sendPostSpamReportMail(network,strPostType,contentId,totalUsers);
+			const postReportKey = await redisGet(`postReportKey-${post_id}`);
+			if(!postReportKey){
+				_sendPostSpamReportMail(network,strPostType,contentId,totalUsers);
+				await redisSetex(`postReportKey-${post_id}`,21600,'true');
+			}
 		}
 		if(type == 'comment' && checkReportThreshold(totalUsers) ){
-			_sendCommentReportMail(network,strPostType,post_id,comment_id,totalUsers);
+
+			const commentReportKey = await redisGet(`commentReportKey-${comment_id}`);
+			if(!commentReportKey){
+				_sendCommentReportMail(network,strPostType,post_id,comment_id,totalUsers);
+				await redisSetex(`commentReportKey-${comment_id}`,21600,'true');
```

Comment:
please avoid using magic numbers and make a const for it

Reply:
resolved